ansible/roles/routemgmt/tasks/clean.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Remove API Gateway route management actions.
+
+- name: remove route management actions
+  shell: ./uninstallRouteMgmt.sh {{ catalog_auth_key }} {{ groups['edge'] | first }} {{ catalog_namespace }} {{ cli_path }} chdir="{{ openwhisk_home }}/ansible/roles/routemgmt/scripts"
+  environment:
+    OPENWHISK_HOME: "{{ openwhisk_home }}"
+  ignore_errors: True
```

Comment:
@lehoanganh - Thanks for taking a look at the ansible files.  This file was modeled after other openwhisk roles (invoker, controller, consul, etc) and most had `ignore errors: True` within their clean.yml ansible files.

My thinking is that routemgmt errors associated with removing non-existent entities are ignored, but other persistent openwhisk failures should be caught during the deploy.yml.